packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -13,6 +13,10 @@
     grid-template-rows: 1fr auto;
     grid-template-columns: auto 1fr;
 
+    &:not(:has(.iui-status-message)) {
+      row-gap: unset;
+    }
```

Comment:
Instead of removing the `row-gap`, would it be better to modify the `grid-template-rows` appropriately? Because I think the real issue is that we're telling it that there are two rows even when there's just one row.

Reply:
Updated to use `grid-template-rows` instead of `row-gap`.